MIT_GEOS5PlugMod/mitgcm_setup/code_split_driver/README.md

```diff
@@ -0,0 +1,5 @@
+MITgcm ocean interface code between GEOS-5 ocean plug and internal MITgcm data structuctures.
```

Comment:
`code between GEOS-5 ocean plug` should be:

code between GEOS ocean plug, **without** `-5`

Reply:
Santha would it be ok to expedite the check-in of all of these comments as is and then update later? i agree that the "-5" should be removed from all of them.

Reply:
No, such an approach **always** defers a graceful treatment and everyone always moves on to other things when they get a ✅. These requested changes are purely **structural** and should not take much of (your) time. I'll convert this PR into a draft and try to find some time to help -- if you need-- for next Tue.

Reply:
OK am changing the comment